### ISSUE TITLE
[FR] Quote/reply to a given message

### ISSUE THREAD

**Issue by breversa** [moezbhatti/qksms#1925]
### DESCRIPTION
Signal (www.signal.org) has the feature to quote/reply to a given message (very useful in long, multi-messages conversations) : 
![image](https://user-images.githubusercontent.com/2071276/219629594-5b063e39-fb5e-4b3c-8ec2-2908cee88121.png)

I understand that the SMS protocol does not provide the capability to do the same ; however, a similar behaviour as reactions (https://github.com/moezbhatti/qksms/issues/1330) could be used.

### EXPECTED
1. Receive an or several SMSs.
2. Swipe right on the SMS you want to quote/reply to.
3. Type your reply and tap "Send".
4. QKSMS will send your reply, preceded or followed by the quoted message (prepended with "In reply to:").

### OBSERVATIONS
This will of course increase the number of sent SMSs, which may incur an additional cost.

**Comment by 87113** [comment]
I love the idea! :) 

I think about another way to do it: 

1. swipe right 
2. QKSMS write the SMS with quote in markdown/email style with > behind each line
3. let the user write is own message quoted SMS or after ( exactly like email actually) 

And anyway, thank you for this app :)

**Comment by e-t-l** [comment]
This would be very difficult to implement. Signal (and Facebook Messenger, emails, Github comments, etc.) can do this because they use a much more complex protocol than SMS. 

SMS is a plaintext format. It does not support markdown. If you write markdown formatting in an SMS, even if *your* messaging client displays it correctly, whomever you are texting will just see the plain symbols. If you want more complex message formatting, switch to Signal or email. If you want to quote-reply someone over SMS, then write it like a human being: "Hey, I remember earlier you said 'XYZ'. My response to that is..."

**Comment by breversa** [comment]
I understand that SMS is a pretty barebones protocol that doesn't allow as much fanciness as Signal.

This feature request aims at mimicking the Signal reply behaviour _within the limitations_ of the SMS protocol, in order the alleviate the hassle of "write it like a human being: "Hey, I remember earlier you said 'XYZ'. My response to that is..."" which is of course what I currently do, and find so cumbersome.

Basically, it boils down to "On right-swipe, copy the text of the swiped message into the text input field (prepended with "In reply to:" or something similar)".
That's it. :)

(I understand that this will very probably often break the character limit per message, but what's the difference between this, and simply typing a very long message?)

**Comment by e-t-l** [comment]
> Basically, it boils down to "On right-swipe, copy the text of the swiped message into the text input field (prepended with "In reply to:" or something similar)".

Hmm to copy a text and paste it into a new message currently requires ~4 taps. This still seems like a niche use case, but I do like the concept of horizontally swiping on a message in the conversation to perform an action. I could easily imagine your case being one of the options for a message-swipe action.

**Comment by breversa** [comment]
> Hmm to copy a text and paste it into a new message currently requires ~4 taps. This still seems like a niche use case, but I do like the concept of horizontally swiping on a message in the conversation to perform an action. I could easily imagine your case being one of the options for a message-swipe action.

4 taps vs 1 swipe is the reason for this PR. :)
Having used it in other messaging apps (ex: Signal), it becomes VERY useful if you have complex, multiple-topics discussions via text messages (which I, and my contacts, have).
